fcore.cabal

```diff
@@ -4,7 +4,7 @@ cabal-version:          >= 1.10
 build-type:             Simple
 license:                BSD3
 license-file:           LICENSE
-copyright:              (c) 2014—2015 The F2J Project Developers (given in AUTHORS.txt)
+copyright:              (c) 2014-2015 The F2J Project Developers (given in AUTHORS.txt)
```

Comment:
Use [em dash](http://en.wikipedia.org/wiki/Dash) instead of hyphen. Any specific reason for having changed this?